extensions/3rdparty/LyricWiki/Hook_PreventBlanking.php

```diff
@@ -3,8 +3,6 @@
 require_once 'extras.php';
 
 $wgHooks['EditFilter'][] = 'stopBlanking';
-$wgAvailableRights[] = 'allowedtoblank';
-$wgGroupPermissions['sysop']['allowedtoblank'] = true;
 $wgExtensionMessagesFiles["PreventBlanking"] = "$LW/Hook_PreventBlanking.i18n.php";
```

Comment:
Not sure I understand - I'm not removing any rights, I'm just moving all definitions to one single place in config.